package.json

```diff
@@ -85,10 +85,8 @@
     "@airbrake/browser": "2.1.3",
     "@hot-loader/react-dom": "17.0.1",
     "axios": "0.21.1",
-    "bluebird": "3.7.2",
```

Comment:
Wasn't this required as a peer dependency of `giantswarm-js-client`?

Reply:
I don't think so... 🕵️ it was introduced here: https://github.com/giantswarm/happa/pull/578/files which was about the original app catalog, but I probably didn't end up using it? Not sure...but as far as I can tell Happa works fine without it.